apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/ColumnQualifier.cs

```diff
@@ -115,6 +119,10 @@ public override string ToString()
         /// <returns></returns>
         public static bool operator !=(ColumnQualifier x, ColumnQualifier y)
         {
+            if (x?.ByteString.Length != y?.ByteString.Length)
```

Comment:
Consider just delegating: `!=(ColumnQualifier x, ColumnQualifier y) => !(x == y)`

Sorry for not suggesting that before.

Reply:
Not a problem. Thanks for pointing that out. I usually do that but just forgot in this case.